docs/try/tutorials/geomodel/geomodel-basics.md

```diff
@@ -1,22 +1,76 @@
+The file extention is the factory key. 
+The Input/Output file format currently supported by RINGMesh are listed [here](/features/file_formats).
+
+A GeoModel can be loaded and saved as follow:
+
+	 // GeoModel instantiation in 3D
+	 GeoModel3D geomodel ;
+	
+	 // LOADING
+	 //path to the file to load
+	 //the file extention is the factroy key.
+	 std::string input_file_name = "path/to/your_input_model.key";
+	 //static function to load a geomodel from input_file_name
+	 geomodel_load( geomodel, input_file_name ) ;
+	 
+	 //SAVING
+	 //path to the file to write
+	 //the file extention is the factroy key.
+	 std::string output_file_name = "path/to/your_output_model.key"; 
+	 geomodel_save(geomodel, output_file_name);
+
+When a GeoModel is loaded, its validity is automatically checked and resulting information 
+are printed in the consol terminal.
+
+####Try:
+
+ * Load the `ModelA1.ml` (a 3D model boundary representation from gocad) 
+ * Save it to the GINGMesh file format (.gm)`ModelA1.gm`.
+ * Open and analyse the .gm file [see description](/features/file_formats).
+
+### Basic Usage
+Once created several functions can be called to study/manipulate/repair the GeoModel:
+
+	print_geomodel_mesh_stats( geomodel ) ; //print the geomodel statistics in the command terminal
+	is_geomodel_valid( geomodel ) ; //investigate the validity of the geomodel
+	translate(geomodel, {0,0,100}) ; //translate geomodel along vertical axis
+	
+If RINGMesh is compilled with a meshing sofware:
+ 
+	tetgen_tetrahedralize_geomodel_regions( geomodel ) ; //build volumetric mesh in geomodel regions using TetGen [Si,2015] 
+	
+This is not an exaustive list of functionalities. Please look at the doxygen documentation.
+
+Try:
+
+ * Load the `ModelA1.ml` (a 3D model boundary representation from gocad) 
+ * print statistics on the imported geomodel
+ * mesh it with TetGen
+ * print statistics on the imported geomodel
+ * Save it to the GINGMesh file format (.gm) `ModelA1.gm`.
+ * Open and analyse the .gm file [see description](/features/file_formats).
  
-The `RINGMesh::GeoModelBuilder` implements functions to build and modify the elements of a `RINGMesh::GeoModel`.
-Any new builder should derive from this class.
-Three main builders are implemented:
+### Manipulate GeoModelEntities
 
- * RINGMesh::GeoModelBuilderGM to load RINGMesh geological model format
-  (boundary representation or 3D meshed models).
- * RINGMesh::GeoModelBuilderGocad to load either:
-    * A SKUA-GOCAD structural model saved in a .ml (RINGMesh::GeoModelBuilderML) file.
-    * A volumetric model saved in a .so (RINGMesh::GeoModelBuilderTSolid) file.
- * RINGMesh::GeoModelBuilderSurfaceMesh and RINGMesh::GeoModelBuilderMesh to build a
-RINGMesh::GeoModel from a RINGMesh::Mesh.
+### Manipulate GeoModelMesh
 
-###GeoModelMesh: How to use the global iteration?
+The GeoModelMesh is contain four data bases to provide a global and unique indexing of mesh component.
```

Comment:
is made of

Reply:
1- The GeoModelMesh contain four data bases to provide a global and unique indexing of mesh component.
or 
2-The GeoModelMesh is made of four data bases to provide a global and unique indexing of mesh component